### PR TITLE
:pencil: (Doctrine) Update docs & package suggestions

### PR DESCRIPTION
**Pull request by feymo**
As discussed in #1344 , before finding a proper way to deal with this issue, i propose to update the documentation & package suggestion.
This way, developpers encoutering this issue will have an easy way to solve it until we implement a more independant solution

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -33,7 +33,6 @@
         "ext-sqlite3": "*",
         "dg/bypass-finals": "^1.3",
         "doctrine/doctrine-bundle": "^2.7",
-        "doctrine/mongodb-odm": "^2.4",
```
Comment (garak) [inline composer.json:36]: this change is unrelated and should be reverted

**Review by garak — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (feymo) [reply composer.json]: Totally, i removed this package by mistake and forgot to put it back... Thanks ! 👍🏼

**Review by garak — CHANGES_REQUESTED** [review]
Please check failures related to the documentation format

**Review by garak — CHANGES_REQUESTED** [review]
I'm not sure if you actually checked the lint errors...
In any case, here they are:
```
docs/known_issues.md:138:103 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/known_issues.md:140:98 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
```

**Review by garak — APPROVED** [review]

**Comment by garak** [comment]
Thank you @feymo !
